core/gui/src/app/workspace/component/menu/menu.component.ts

```diff
@@ -253,17 +253,33 @@ export class MenuComponent implements OnInit {
    * get the html to export all results.
    */
   public onClickGenerateReport(): void {
-    //Get notification
+    // Get notification
     this.notificationService.info("The report is being generated...");
 
     const workflowName = this.currentWorkflowName;
+    const workflowContent: WorkflowContent = this.workflowActionService.getWorkflowContent();
+    const workflowContentJson = JSON.stringify(workflowContent);
+    const payload = JSON.parse(workflowContentJson);
```

Comment:
you don't need to stringify first then parse, right? You can directly work on the `workflowContent`